fe/fe-core/src/main/java/org/apache/doris/load/routineload/RoutineLoadManager.java

```diff
@@ -483,8 +484,10 @@ public RoutineLoadJob getJob(String dbFullName, String jobName) throws MetaNotFo
       if includeHistory is false, filter not running job in result
       else return all of result
      */
-    public List<RoutineLoadJob> getJob(String dbFullName, String jobName, boolean includeHistory)
+    public List<RoutineLoadJob> getJob(String dbFullName, String jobName, boolean includeHistory, PatternMatcher matcher)
             throws MetaNotFoundException {
+        Preconditions.checkArgument(jobName == null || matcher == null,
```

Comment:
can be both null, but not both not null ~

Reply:
this check mean that jobName and matcher should not both not null at the same time

Reply:
you are right ~